main/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+class Event(models.Model):
+    title = models.CharField(max_length=100)
+    date = models.DateField()
+    start_time = models.TimeField()
```

Comment:
It would be great to have a requirement that start_time should be earlier than end_time

Reply:
I will look into this bug and try to fix it.